deploy/roles/sqitch/tasks/main.yml

```diff
@@ -1,13 +1,8 @@
 ---
 - name: Install dependencies
   yum:
-    name: "{{ item }}"
+    name: ['perl-devel', 'perl-CPAN', '@Development Tools'] # perl-DBD-Pg
     state: present
-  with_items:
-    - perl-devel
-    - perl-CPAN
-    #- perl-DBD-Pg
-    - "@Development Tools"
   tags:
     - sqitch
 
```

Comment:
It turns out that newer versions of Sqitch have changes in how the command line works, so we need to pin to the current version.  Can you replace the line `shell: cpanm --notest --sudo App::Sqitch` with `shell: cpanm --notest --sudo DWHEELER/App-Sqitch-0.9996.tar.gz`